Neuro/Tasks/TasksHandler.cs

```diff
@@ -174,28 +174,31 @@ private void Update()
         if (Minigame.Instance) return;
         if (!PlayerControl.LocalPlayer) return;
 
-        foreach (PlayerTask task in PlayerControl.LocalPlayer.myTasks)
+        //Open anything nearby, speedrun.
+        foreach (var tasky in PlayerControl.LocalPlayer.myTasks)
         {
-            if (task.IsComplete || task.Locations == null) continue;
+            if (tasky.IsComplete || tasky.Locations == null) continue;
 
-            // TODO: HOLY FUCK DON'T UNCOMMENT THIS THIS IS BROKEN AF
-            // INVOKE CONSOLE.USE DIRECTLY PLEEEEEEASE
+            Il2CppSystem.Collections.Generic.List<Console> list = tasky.FindConsoles();
 
-            /*// TODO: Invoke Console.Use directly and check if we can open it before trying
-            foreach (Vector2 location in task.Locations)
-            {
-                if (Vector2.Distance(location, PlayerControl.LocalPlayer.transform.position) < 0.8f)
+            foreach (Console con in list) {
+                var currentDistance = Vector2.Distance(con.transform.position, PlayerControl.LocalPlayer.transform.position);
+
+                if (currentDistance < con.usableDistance && !con.name.Contains("Vent"))
                 {
-                    // TODO: Check if we should open the task with the MinigameSolver before actually opening it
-                    if (task.MinigamePrefab)
+                    // Info("We found " + con.name + " with distance " + con.usableDistance + " " + currentDistance);
+                    bool couldUse;
+                    bool canUse;
+                    con.CanUse(PlayerControl.LocalPlayer.Data, out canUse, out couldUse);
```

Comment:
canUse can be inlined with `out bool canUse` and we don't need couldUse so it can just be discarded `out _`

Reply:
> canUse can be inlined with `out bool canUse` and we don't need couldUse so it can just be discarded `out _`

Fixed, never used  c# before, so thanks for the tips.